### PR TITLE
[IMP] l10n_ec: Improvements in tax group accounts to automate tax closing entry

### PR DESCRIPTION
**Pull request by stevTresCloud**
- These new accounts are to automate the tax closing entry on 103 and 104 reports.
- The tax names of account 11050201 is changed to a better accounting standard name for the understanding of the accounting accounts to be configured in the tax groups, based on the name of the new account group created.
- Accounts payable and receivable are added for each of the tax groups, by company. It's not set in chart template property accounts, because they are different accounts for each tax group. For this reason, the accounts are set in load chart template method.
- Add irbpnr tax group account
- Add Exchange Outflows accounts and fix ec translation for exchange outflows account

Description of the issue/feature this PR addresses:
The tax account entry in 103 and 104 reports is not working yet, because the payable and receivable accounts are not set in each tax group.

Current behavior before PR:
The tax account entry in 103 and 104 reports is not working yet, because the payable and receivable accounts are not set in each tax group.

Desired behavior after PR is merged:
Automate the tax closing entry on 103 and 104 reports. This entry must be edited by the company's accountant.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ec/data/account.account.template.csv [hunk 1]
```diff
@@ -114,6 +118,9 @@ ec21050102,21050102,Otras provisiones locales,TRUE,liability_current,l10n_ec_ifr
 ec210502,210502,Provisiones del exterior,TRUE,liability_current,l10n_ec_ifrs
 ec2106,2106,Porcion corriente de obligaciones emitidas,FALSE,liability_current,l10n_ec_ifrs
 ec21070101,21070101,Impuesto a la renta retenido a empleados,TRUE,liability_current,l10n_ec_ifrs
+ec21070102,21070102,Impuesto por pagar IVA,TRUE,liability_current,l10n_ec_ifrs
+ec21070103,21070103,Retenciones en la fuente por pagar,TRUE,liability_current,l10n_ec_ifrs
+ec21070104,21070104,Impuesto ICE por pagar,TRUE,liability_current,l10n_ec_ifrs
```
Comment (pepetreshere) [inline addons/l10n_ec/data/account.account.template.csv:123]: Maybe you need to add an "others" account?

File: addons/l10n_ec/data/account.account.template.csv [hunk 2]
```diff
@@ -36,7 +36,11 @@ ec_purchase_vat_goods_imports,11050104,IVA pagado en importacion de bienes,FALSE
 ec_purchase_vat_assets_imports,11050105,IVA pagado en importacion de activos fijos,FALSE,asset_current,l10n_ec_ifrs
```
Comment (pepetreshere) [inline addons/l10n_ec/data/account.account.template.csv:36]: please include the accounts for the plastic bottles so that we have one single PR modifying the chart of accounts

**Review by pepetreshere — CHANGES_REQUESTED** [review]

File: addons/l10n_ec/data/account.account.template.csv (hunk 1 above)
Comment (stevTresCloud) [reply addons/l10n_ec/data/account.account.template.csv:123]: @stevTresCloud  Add missing account "Otros impuestos por pagar"

File: addons/l10n_ec/data/account.account.template.csv (hunk 2 above)
Comment (stevTresCloud) [reply addons/l10n_ec/data/account.account.template.csv:36]: Ok. It's ready.

File: addons/l10n_ec/data/account.account.template.csv [hunk 3]
```diff
@@ -114,6 +118,10 @@ ec21050102,21050102,Otras provisiones locales,TRUE,liability_current,l10n_ec_ifr
 ec210502,210502,Provisiones del exterior,TRUE,liability_current,l10n_ec_ifrs
 ec2106,2106,Porcion corriente de obligaciones emitidas,FALSE,liability_current,l10n_ec_ifrs
 ec21070101,21070101,Impuesto a la renta retenido a empleados,TRUE,liability_current,l10n_ec_ifrs
+ec21070102,21070102,Impuesto por pagar IVA,TRUE,liability_current,l10n_ec_ifrs
+ec21070103,21070103,Retenciones en la fuente por pagar,TRUE,liability_current,l10n_ec_ifrs
+ec21070104,21070104,Impuesto ICE por pagar,TRUE,liability_current,l10n_ec_ifrs
```
Comment (pepetreshere) [inline addons/l10n_ec/data/account.account.template.csv:123]:
```suggestion
ec21070104,21070104,Impuesto por pagar ICE,TRUE,liability_current,l10n_ec_ifrs
```

**Review by pepetreshere — CHANGES_REQUESTED** [review]

**Comment by vbe-odoo** [comment]
Hi @stevTresCloud 
When trying to install the EC modules I am receiving the following traceback:
image

Can you please help me check on this?
Thanks!
cc' @pepetreshere @fvz-odoo @ren-odoo

**Comment by stevTresCloud** [comment]
@vbe-odoo it's already fixed. Please, do again the FUN review.

File: addons/l10n_ec/data/account.account.template.csv [hunk 4]
```diff
@@ -113,7 +117,11 @@ ec2105010104,2105010104,Otras provisiones a favor de los empleados,TRUE,liabilit
 ec21050102,21050102,Otras provisiones locales,TRUE,liability_current,l10n_ec_ifrs
 ec210502,210502,Provisiones del exterior,TRUE,liability_current,l10n_ec_ifrs
 ec2106,2106,Porcion corriente de obligaciones emitidas,FALSE,liability_current,l10n_ec_ifrs
-ec21070101,21070101,Impuesto a la renta retenido a empleados,TRUE,liability_current,l10n_ec_ifrs
```
Comment (jco-odoo) [inline addons/l10n_ec/data/account.account.template.csv:116]: please avoid unnecessary change here of the xml_id

Comment (vbe-odoo) [reply addons/l10n_ec/data/account.account.template.csv:116]: @stevTresCloud @pepetreshere

**Review by pepetreshere — CHANGES_REQUESTED** [review]

**Comment by stevTresCloud** [comment]
@pepetreshere This is a screenshot with the tax groups and the configuration of the accounts for the closing entry.
image

**Comment by pepetreshere** [comment]
@stevTresCloud I will review again

**Comment by pepetreshere** [comment]
@jco-odoo this PR is ready for your review!

**Comment by vbe-odoo** [comment]
Thanks for the changes @pepetreshere and @stevTresCloud .
I've tested and LGMT to me.
Ready for your review @jco-odoo 
Thanks!
cc' @ren-odoo @fvz-odoo

File: addons/l10n_ec/models/account_chart_template.py [hunk 5]
```diff
@@ -7,6 +7,45 @@
...
+            ('tax_group_irbpnr', '21070105', '21070105'),
+            ('tax_group_withhold_vat_sale', '21070102', '11050203'),
+            ('tax_group_withhold_vat_purchase', '21070102', '11050203'),
+            ('tax_group_withhold_income_sale', '21070103', '11050201'),
+            ('tax_group_withhold_income_purchase', '21070103', '11050201'),
+            ('tax_group_outflows', '21070106', '11050205'),
+            ('tax_group_others', '21070106', '11050205'),
+        ]
+        for tax_group_xml_id, payable_account_code, receivable_account_code in _TAX_GROUPS_ACCOUNTS_LIST:
+            for company in companies.filtered(lambda company: company.account_fiscal_country_id.code == 'EC'):
```
Comment (jco-odoo) [inline addons/l10n_ec/models/account_chart_template.py:39]: here you should check the Chart template instead because you do not know if the CoA is installed in that company

**Comment by vbe-odoo** [comment]
FUN test was done and LGTM.
Ready for tech review @jco-odoo 
cc' @pepetreshere @stevTresCloud @ren-odoo @fvz-odoo

File: addons/l10n_ec/data/account_group_template_data.xml [hunk 6]
```diff
@@ -122,6 +122,13 @@
             <field name="parent_id" ref="ec1105"/>
         </record>
 
+        <record id="ec110502" model="account.group.template">
+            <field name="chart_template_id" eval="ref('l10n_ec.l10n_ec_ifrs')"/>
```
Comment (jco-odoo) [inline addons/l10n_ec/data/account_group_template_data.xml:126]:
```suggestion
            <field name="chart_template_id" ref="l10n_ec.l10n_ec_ifrs"/>
```

**Review by jco-odoo — COMMENTED** [review]
just a minor thing

**Comment by jco-odoo** [comment]
@robodoo r+ rebase-ff
